app/controllers/concerns/idv_step_concern.rb

```diff
@@ -17,7 +17,7 @@ module IdvStepConcern
   end
 
   def confirm_no_pending_gpo_profile
-    redirect_to idv_gpo_verify_url if current_user&.gpo_verification_pending_profile?
+    redirect_to idv_enter_verify_by_mail_code_url if current_user&.gpo_verification_pending_profile?
```

Comment:
One more suggestion if you can stand to do another search/replace. Every time I read this I think it's entering the verify by mail flow.
```suggestion
    redirect_to idv_verify_by_mail_enter_code_url if current_user&.gpo_verification_pending_profile?
```

Reply:
Yeah I like that variation better, I will update